x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/transform_table.tsx

```diff
@@ -23,7 +23,9 @@ export const ProgressBar = ({ isLoading = false }) => {
   return (
     <Fragment>
       {isLoading && <EuiProgress size="xs" color="primary" />}
-      {!isLoading && <EuiProgress value={0} max={100} size="xs" />}
+      {!isLoading && (
+        <EuiProgress className="mlTransformProgressBar" value={0} max={100} size="xs" />
```

Comment:
You also need to add `className="mlTransformProgressBar"` to the loading state of the `EuiProgress` element on the line above to prevent the content 'jumping' up when `isLoading===true`

Reply:
Added in 6767eee8ea73293cd295e2c9af9ef59408269642